Makefile

```diff
@@ -15,6 +15,10 @@ LIB_CRYSTAL_SOURCES = $(shell find src/ext -name '*.c')
 LIB_CRYSTAL_OBJS = $(subst .c,.o,$(LIB_CRYSTAL_SOURCES))
 LIB_CRYSTAL_TARGET = src/ext/libcrystal.a
 
+ifeq (${LLVM_CONFIG},)
+$(error Could not locate LLVM Config binary)
```

Comment:
I'd elaborate a bit, perhaps "Could not locate llvm-config, make sure it's installed and in your $PATH"